Authentication/LoginRecord.csv

```diff
@@ -1 +1,7 @@
 Name,Time,Date
+2019-CE-34,22:55:31,11-12-21
+2019-CE-34,22:58:22,11-12-21
+WIN_20211212_00_18_34_Pro,00:19:31,12-12-21
+2019-CE-41,00:22:46,12-12-21
+2019-CE-41,16:16:14,12-12-21
+2019-CE-41,16:17:12,12-12-21
```

Comment:
### Changes Required...
- Only the first line was required...
- Push it with that commit...